src/main/java/seedu/address/logic/commands/ArchiveCommand.java

```diff
@@ -20,7 +20,7 @@ public class ArchiveCommand extends Command {
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_ARCHIVE_PERSON_SUCCESS = "%1$s has been archived. Type “alist” to "
+    public static final String MESSAGE_ARCHIVE_PERSON_SUCCESS = "Archived Person: %1$s.\nType “alist” to "
```

Comment:
Editing the message success so that it follows the previous success messages more closely is a great idea